### PR TITLE
Prints GIS types into a kml file

### PR DESCRIPTION
**Pull request by raychen323**

### CODE REVIEW

File: src/main/java/com/ocient/cli/CLI.java [hunk 1]
```diff
@@ -1549,6 +1568,168 @@ else if (o != null)
...
+
+		//generate 2nd gx cascading style
+		Element gx2 = doc.createElement("gx:CascadingStyle");
+		documentElement.appendChild(gx2);
+		Attr gx2ID = doc.createAttribute("kml:id");
+		gx2ID.setValue("__managed_style_2C97020CD31971D9081A");
+		gx2.setAttributeNode(gx2ID);
+		Element gx2Style = doc.createElement("Style");
+		gx2.appendChild(gx2Style);
+
```
Comment (leogreca) [inline src/main/java/com/ocient/cli/CLI.java:1649]: Add some comments on how the style is set.

File: src/main/java/com/ocient/cli/CLI.java [hunk 2]
```diff
@@ -1469,6 +1487,7 @@ private static void outputNextQuery(final String cmd)
 
 	private static void outputResultSet(final ResultSet rs, final ResultSetMetaData meta) throws Exception
 	{
+		outputGeospatial(rs, meta);
 		final FileWriter fw = new FileWriter(outputCSVFile);
```
Comment (tli-ocient) [inline src/main/java/com/ocient/cli/CLI.java:1491]: How does this work? Does this happen every time?

**Review by leogreca — APPROVED** [review]

File: src/main/java/com/ocient/cli/CLI.java (hunk 2 above)
Comment (raychen323) [reply src/main/java/com/ocient/cli/CLI.java:1491]: Happens every time

Comment (raychen323) [reply src/main/java/com/ocient/cli/CLI.java:1491]: It only outputs GIS types to the kml though

Comment (leogreca) [reply src/main/java/com/ocient/cli/CLI.java:1491]: It happens all the time according to Raymond. I mean the kml will always be generated. I kind of don't see a problem with that. Do you?

Comment (tli-ocient) [reply src/main/java/com/ocient/cli/CLI.java:1491]: I would rather you not put this here. Make a new cli command for it. "output geospatial kml" or something.

File: src/main/java/com/ocient/jdbc/StPolygon.java [hunk 3]
```diff
@@ -54,4 +58,55 @@ public String toString()
 
         return str.toString();
 	}
+
+    public void writeXML(Document doc, Element docElement, String name)
+	{
+        Element placemark = doc.createElement("Placemark");
+		docElement.appendChild(placemark);
+
```
[the remark this answers is not in the document]
Comment (raychen323) [reply src/main/java/com/ocient/jdbc/StPolygon.java:66]: it's not showing locally

**Review by tli-ocient — APPROVED** [review]

**Pull request merged**